### PR TITLE
fixed other petitioners display to not look broken when no secondary name is supplied

### PR DESCRIPTION
**Pull request by mrinsin**
image

### CODE REVIEW

File: web-client/src/views/CaseDetail/OtherPetitionerDisplay.jsx [hunk 1]
```diff
@@ -13,8 +13,13 @@ const OtherPetitionerDisplay = connect(
       <>
         <p className="no-margin address-name">
           {petitioner.name}
-          {petitioner.secondaryName && <div>{petitioner.secondaryName}</div>}
-          {petitioner.title && <span>, {petitioner.title}</span>}
+          {petitioner.secondaryName && (
+            <>
+              <br />
+              {petitioner.secondaryName}
+              {petitioner.title && <span>, {petitioner.title}</span>}
```
Comment (matthopson) [inline web-client/src/views/CaseDetail/OtherPetitionerDisplay.jsx:20]: So we _only_ show the title when they have a secondary name?

Comment (mrinsin) [reply web-client/src/views/CaseDetail/OtherPetitionerDisplay.jsx:20]: I believe so, as explained to me by @ericsorenson !

**Comment by matthopson** [comment]
Is the screenshot accurate? I just changed this view to remove the gap between the petitioner name and address per a `ux pain point` task.

**Review by matthopson — APPROVED** [review]

**Comment by mrinsin** [comment]
@matthopson the screenshot was outdated, just edited it!

**Pull request merged**